lighthouse-core/scripts/open-devtools.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+##
+# @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+##
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+LH_ROOT="$SCRIPT_DIR/../.."
+TEST_DIR="$LH_ROOT/.tmp/chromium-web-tests"
+DEFAULT_DEVTOOLS_PATH="$TEST_DIR/devtools/devtools-frontend"
+DEVTOOLS_PATH=${DEVTOOLS_PATH:-"$DEFAULT_DEVTOOLS_PATH"}
+
+if [[ -z $CHROME_PATH ]]; then
+  echo 'Must set $CHROME_PATH'
+  exit 1
+fi
+
+echo "CHROME_PATH: $CHROME_PATH"
+echo "DEVTOOLS_PATH: $DEVTOOLS_PATH"
+
+if [ ! -d $DEVTOOLS_PATH ]; then
+  echo "No devtools found at $DEVTOOLS_PATH. Have you run 'yarn test-devtools' yet?"
+  exit 1
+fi
+
```

Comment:
I had to add this path setup from
https://github.com/GoogleChrome/lighthouse/blob/d4fc00cec56b80924c6f661922f1092ee25734e6/lighthouse-core/test/chromium-web-tests/test-locally.sh#L16-L19 since I don't leave depot_tools on my path when not in chromium dirs


```suggestion

if ! which gn ; then
  # If the contributor doesn't have a separate depot tools in their path, use the tmp copy.
  DEPOT_TOOLS_PATH="$TEST_DIR/depot-tools"
  BLINK_TOOLS_PATH="$TEST_DIR/blink_tools"
  export PATH=$DEPOT_TOOLS_PATH:$PATH
  # Add typ to python path. The regular method assumes there is a Chromium checkout.
  export PYTHONPATH="${PYTHONPATH:-}:$BLINK_TOOLS_PATH/latest/third_party/typ"
fi

```

Reply:
Why the python path? We aren't running web_tests.py or w.e. it's called.

Reply:
not sure exactly what part of the dependency toolchain requires it, but this is the error you get if you don't have it in your path after calling `gclient sync`

```
Error: Command 'vpython third_party/depot_tools/update_depot_tools_toggle.py --disable' returned non-zero exit status -9 in /Users/patrick/Code/OpenSource/lighthouse/.tmp/chromium-web-tests/devtools/devtools-frontend
```